x-pack/plugins/actions/server/lib/get_token_count_from_invoke_stream.ts

```diff
@@ -95,16 +99,33 @@ const parseBedrockStream: StreamParser = async (responseStream, logger) => {
   return parseBedrockBuffer(responseBuffer);
 };
 
-const parseOpenAIStream: StreamParser = async (responseStream, logger) => {
+const parseOpenAIStream: StreamParser = async (responseStream, logger, signal) => {
   let responseBody: string = '';
-  responseStream.on('data', (chunk) => {
+  const destroyStream = () => {
+    // Pause the stream to prevent further data events
```

Comment:
Yeah I did this to get jest to pass. Without it, `getTokenCountFromInvokeStream › OpenAI stream › Stops the stream early when the request is aborted` will fail because the stream didn't get destroyed :(

Reply:
I'm sure I found it on stackoverflow somewhere but cannot find it now